### PR TITLE
Docs: minor improvements to i18n docs, mention French

### PR DESCRIPTION
**Pull request by rubo77**

### CODE REVIEW

File: docs/dev/i18n.rst [hunk 1]
```diff
@@ -4,8 +4,8 @@ Internationalization support
 General guidelines
 ------------------
 
-* All config mode packages must be fully translatable, with complete English and German texts.
-* All new expert mode packages be fully translatable. English texts are required, German texts recommended.
+* All config mode packages must be fully translatable, with complete English and German texts, French texts recommended
+* All new expert mode packages be fully translatable. English texts are required, German and French texts recommended.
```
Comment (NeoRaider) [inline docs/dev/i18n.rst]: Maybe: "Translations for other supported languages are nice-to-have, but not required. If you don't know a language well, rather leave the translation blank, so it is obvious that there is no proper translation yet."

File: docs/dev/i18n.rst [hunk 2]
```diff
@@ -84,5 +84,5 @@ A list of all languages supported by LuCI can be found in the ``packages/luci/lu
 Gluon's dependencies have been downloaded using ``make update``. Adding translations for these
 languages is straightforward using the ``msginit`` command.
 
-For other languages, support must be added tu LuCI first, which constitutes completely translating
-the ``base.pot``. Please contact the upstream LuCI maintainers if you'd like to do this.
+For other languages, support must be added to LuCI first, which constitutes completely translating
+the ``base.pot``. Please contact the upstream LuCI maintainers if you'd like to do this at https://github.com/openwrt/luci/ .
```
Comment (NeoRaider) [inline docs/dev/i18n.rst]: The ordering is wrong, the sentence should be "Please contact the upstream LuCI maintainers at https://github.com/openwrt/luci/ if you'd like to do this"

**Comment by NeoRaider** [comment]
The commit title doesn't match the changes, maybe just "docs: minor improvements to i18n docs"?

**Comment by rubo77** [comment]
done

File: docs/dev/i18n.rst [hunk 3]
```diff
@@ -5,7 +5,10 @@ General guidelines
 ------------------
 
 * All config mode packages must be fully translatable, with complete English and German texts.
-* All new expert mode packages be fully translatable. English texts are required, German texts recommended.
+* All new expert mode packages must be fully translatable. English texts are required.
+* Translations for German tests are recommended. Other supported languages, especially french are
```
Comment (NeoRaider) [inline docs/dev/i18n.rst]: Typo: "tests", also the sentence doesn't make sense like this. "German translations are recommended" would be clearer.

"French" must be capitalized. A comma is missing after "French".

**Comment by rubo77** [comment]
amended!

**Pull request merged**